src/core/classes/erc-721.ts

```diff
@@ -72,6 +75,7 @@ export class Erc721<
     this.query = this.detectErc721Enumerable();
     this.mint = this.detectErc721Mintable();
     this.drop = this.detectErc721Dropable();
+    this.signatureMint = this.detectErc721SignatureMintable();
```

Comment:
let's call this `sig` instead. so the API looks like `contract.sig.generate()` and `contract.sig.mint()`

and keep the same name accross ERC20/721/1155